vendor/github.com/openshift/library-go/pkg/operator/configobserver/network/observe_network.go

```diff
@@ -3,7 +3,7 @@ package network
 import (
```

Comment:
This change needs to be made in library-go but you don't seem to have an outstanding PR there?

Reply:
was waiting for the API change to merge, but yes.